torchao/quantization/quantize_/workflows/packing_format.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
actually is this the right place for this file? It's not really a "workflow", more like a "util class" or "common"